File: src/main/java/me/sd_master92/customvoting/commands/voteparty/VotePartyCreateCommand.java
package me.sd_master92.customvoting.commands.voteparty;

import me.sd_master92.customvoting.subjects.VoteParty;
import me.sd_master92.plugin.command.SimpleSubCommand;
import org.bukkit.ChatColor;
import org.bukkit.command.CommandSender;
import org.bukkit.entity.Player;

public class VotePartyCreateCommand extends SimpleSubCommand
{
    public VotePartyCreateCommand()
    {
        super("create");
        withPlayer();
        withPermission("voteparty.create");
    }

    @Override
    public void onCommand(CommandSender sender, String[] args)
    {

    }

    @Override
    public void onCommand(Player player, String[] args)
    {
        player.getInventory().addItem(VoteParty.VOTE_PARTY_ITEM);
        player.sendMessage(ChatColor.GREEN + "You have been given the Vote Party Chest.");
    }
}

File: src/main/kotlin/me/sd_master92/customvoting/commands/voteparty/VotePartyCreateCommand.kt
package me.sd_master92.customvoting.commands.voteparty

import me.sd_master92.core.command.SimpleSubCommand
import me.sd_master92.customvoting.CV
import me.sd_master92.customvoting.gui.pages.overviews.VotePartyChestOverview
import org.bukkit.command.CommandSender
import org.bukkit.entity.Player

class VotePartyCreateCommand(private val plugin: CV) : SimpleSubCommand("create")
{
    override suspend fun onCommand(sender: CommandSender, args: Array<out String>)
    {
    }

    override suspend fun onCommand(player: Player, args: Array<out String>)
    {
        VotePartyChestOverview(plugin, null).open(player)
    }

    init
    {
        withPlayer()
        withPermission("voteparty.create")
    }
}